testsuite/.rubocop_todo.yml

```diff
@@ -394,39 +392,24 @@ Style/DocumentationMethod:
     - 'features/support/namespaces/system.rb'
     - 'features/support/twopence_init.rb'
 
-# Offense count: 38
+# Offense count: 39
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: always, always_true, never
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-# Offense count: 950
+# Offense count: 976
 # Configuration parameters: AllowedVariables.
 Style/GlobalVars:
-  Exclude:
-    - 'features/step_definitions/api_common.rb'
-    - 'features/step_definitions/command_steps.rb'
-    - 'features/step_definitions/common_steps.rb'
-    - 'features/step_definitions/docker_steps.rb'
-    - 'features/step_definitions/file_management_steps.rb'
-    - 'features/step_definitions/navigation_steps.rb'
-    - 'features/step_definitions/retail_steps.rb'
-    - 'features/step_definitions/salt_steps.rb'
-    - 'features/step_definitions/setup_steps.rb'
-    - 'features/step_definitions/smdba_steps.rb'
-    - 'features/support/commonlib.rb'
-    - 'features/support/env.rb'
-    - 'features/support/namespaces/system.rb'
-    - 'features/support/twopence_init.rb'
-    - 'features/support/xmlrpc_client.rb'
+  Enabled: false
 
 # Offense count: 16
 # Cop supports --auto-correct.
 Style/IfUnlessModifier:
```

Comment:
We have this clause plenty of exclusions, maybe it's also time to just set it as `Enabled: false`